packages/node/src/sdk.ts

```diff
@@ -152,3 +144,33 @@ export async function close(timeout?: number): Promise<boolean> {
   }
   return Promise.reject(false);
 }
+
+/**
+ * A function that returns a Sentry release string dynamically from env variables
+ */
+export function getSentryRelease(): string {
```

Comment:
This shouldn't be exported though. No need to.

Reply:
Fixed thanks! :)